### ISSUE TITLE
Cant mount a ntfs drive in thunar with boot.supportedFileSystems = ["nfts"]; enabled

### ISSUE THREAD

**Issue by Tofixrs** [NixOS/nixpkgs#243234]
### Describe the bug
Cant mount a ntfs drive in thunar with boot.supportedFileSystems = ["nfts"]; enabled

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.38, NixOS, 23.11 (Tapir), 23.11pre503276.f2406198ea0`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.15.1`
 - channels(root): `"nixos"`
 - channels(tofix): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by Tofixrs** [comment]
the screenshot was all black gonna add the error in a sec

**Comment by Tofixrs** [comment]
![image](https://github.com/NixOS/nixpkgs/assets/73693639/9a7ab5d0-79a0-4dd3-b99a-dec99e99d2eb)

**Comment by surfaceflinger** [comment]
You want ntfs, not nfts if what you said has been copied straight from your config :)

**Comment by Tofixrs** [comment]
> You want ntfs, not nfts if what you said has been copied straight from your config :)

nah just mistyped it here
also i can mount the drive with the mount command

**Comment by SuperSamus** [comment]
Some random things and thoughts:

- Linux has a ntfs3 driver built-in since 5.15.
  - Note that, for mounting with GUIs like Thunar, the newest version of udisks (2.10.0) is required.
  - Have you tried without `boot.supportedFileSystems = ["ntfs"];`? (The only thing it does is adding `ntfs-3g`, a user space driver)
- When mounting from the terminal, is it mounted with `ntfs` (the old kernel driver, read-only), `ntfs3` (the new kernel driver), or `ntfs-3g` (the userspace driver)?

**Comment by Tofixrs** [comment]
>     * When mounting from the terminal, is it mounted with `ntfs` (the old kernel driver, read-only), `ntfs3` (the new kernel driver), or `ntfs-3g` (the userspace driver)?
How do i check?

**Comment by Tofixrs** [comment]
>       * Have you tried without `boot.supportedFileSystems = ["ntfs"];`? (The only thing it does is adding `ntfs-3g`, a user space driver)
Yeah it doesnt work

**Comment by SuperSamus** [comment]
> How do i check?

Simply with `mount`.

> Without it the mount command doesnt mount too

I checked the Arch wiki.
Without `-t`, it tries `ntfs-3g`, then `ntfs`. If you want `ntfs3` it must be used explicitly with `mount -t ntfs3`.

> Yeah it doesnt work

I just tried myself, two ntfs drives mounted no problem with Thunar. (With `ntfs3`)
Used latest `nixpkgs-unstable`, with:

```nix
programs.thunar.enable = true;
programs.thunar.plugins = with pkgs.xfce; [
  thunar-volman
];
services.gvfs.enable = true;
```

So... I don't know, maybe try `sudo ntfsfix -bd /dev/sda1`? (Replace `/dev/sda1` with appropriate path)
